data/spells/scripts/support/find_person.lua

```diff
@@ -46,12 +46,7 @@ function onCastSpell(creature, variant)
 	end
 
 	local playerPosition = creature:getPosition()
-	local targetPosition = targetPlayer:getPosition()
-	local offset = {
-		x = playerPosition.x - targetPosition.x,
-		y = playerPosition.y - targetPosition.y,
-		z = playerPosition.z - targetPosition.z
-	}
+	local offset = playerPosition - targetPlayer:getPosition()
```

Comment:
Seems like it wraps around since x, y and z are unsigned, you can go ahead and revert it then.